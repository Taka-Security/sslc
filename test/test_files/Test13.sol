contract A {
  enum EnumSmall { zero, one, two, three }
  
  struct StructA {
    uint248 var1;
    uint256 var2;
    EnumSmall var3;
  }
}