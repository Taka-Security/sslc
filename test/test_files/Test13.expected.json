[
    {
        "file": "Test13.sol",
        "contract": "A",
        "struct": "StructA",
        "slotsSaved": 1,
        "timedout": false
    }
]